F1WM.UnitTests/Controllers/NewsControllerTests.cs

```diff
@@ -79,5 +79,92 @@ public async Task ShouldReturnEmptyListOfImportantNews()
 			serviceMock.Verify(s => s.GetImportantNews(), Times.Once);
 			Assert.Empty(result);
 		}
-	}
+
+        [Fact]
+        public async Task ShouldReturnNewsTypes()
+        {
+            await controller.GetTypes();
+
+            serviceMock.Verify(s => s.GetNewsTypes(), Times.Once);
+        }
+
+        [Fact]
+        public async Task ShouldReturnNewsTags()
+        {
+            await controller.GetTags();
+
+            serviceMock.Verify(s => s.GetNewsTags(), Times.Once);
+        }
+
+        [Fact]
+        public async Task ShouldReturnNewsCategories()
+        {
+            await controller.GetCategories();
+
+            serviceMock.Verify(s => s.GetNewsCategories(), Times.Once);
+        }
+
+        [Fact]
+        public async Task ShouldReturnNewsTagsByCategory()
+        {
+            var id = 1;
+
+            await controller.GetTagsbyCategory(id);
+
+            serviceMock.Verify(s => s.GetNewsTagsByCategory(id), Times.Once);
+        }
+
+        [Fact]
+        public async Task ShouldReturnNewsByType()
+        {
+            var id = 0;
```

Comment:
Although not harmful in this case, I'd recommend to not use default values as test data.